providers/ethereum-provider/package.json

```diff
@@ -50,22 +50,14 @@
     "@walletconnect/sign-client": "2.10.4",
     "@walletconnect/types": "2.10.4",
     "@walletconnect/universal-provider": "2.10.4",
+    "@walletconnect/modal": "^2.4.3",
```

Comment:
why this change?

Reply:
the modal is moved to an actual dependency so its always downloaded togheder with Ethereum provider thus resolving the build issues

Reply:
will this bring versioning issues with other packages that use another version of the modal? Wagmi for example has a fixed version the modal -> [here](https://github.com/wagmi-dev/wagmi/blob/eb06c91b6b1b27c6b8283060bdbd6f013bbf86ef/packages/connectors/package.json#L27)

build issues doesn't solve if people install the modal dependency following our docs?
`yarn add @walletconnect/ethereum-provider @walletconnect/modal`

Reply:
Any patch version would be fine